libs/isograph-react/src/core/read.ts

```diff
@@ -343,9 +343,12 @@ function readData<TReadFromStore>(
         } else {
           const refetchQueryIndex = field.refetchQuery;
           if (refetchQueryIndex == null) {
```

Comment:
oops, this can never be null... the field is not optional. This is probably a holdover from a previous time. perhaps something like https://typescript-eslint.io/rules/no-unnecessary-condition/ can prevent this in the future